docs/parameters.md

```diff
@@ -481,11 +538,16 @@ Notes:
 
 #### Events output
 
-SIPNET will create a file named `events.out` when event handling is enabled. This file 
-will have one row for each agronomic event that is processed. Each row lists location, 
+SIPNET will create a file named `events.out` when event handling is enabled. 
+
+_Note: `events.out` is primarily intended as a log debugging and testing purposes; 
+the `events.in` contains the same information but is easier to parse for downstream analyses._
```

Comment:
I wouldn't call it the same information for events that calculate changes in other parameters beyond what's specified in the events.in line, e.g. tillage decomposition multipliers

```suggestion
_Note: `events.out` logs all parameters changed by an event for debugging and testing purposes; 
For downstream analyses that only need the date and event type, `events.in` is equivalent and easier to parse._
```